uint8 mode # This sets the operating mode of wheeled_robin 
---
bool valid_mode # will return false if a unvalid mode was requested
